HousingRegisterApi.Tests/V1/Services/BedroomCalculationServiceTests.cs

```diff
@@ -0,0 +1,510 @@
+using HousingRegisterApi.V1.Domain;
+using HousingRegisterApi.V1.Services;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HousingRegisterApi.Tests.V1.Services
+{
+    [TestFixture]
+    public class BedroomCalculationServiceTests
+    {
+        private const string Male = "male";
```

Comment:
Stored values for gender are `M` and `F` and `self`. See input options in the housing-register app. Ideally this would be defined as an enum or lookup.

Reply:
Ah ok.. i will replace